### PR TITLE
Ebony strings

### PR DESCRIPTION
**Pull request by boek**
@flodolo @Delphine 

This is the first export using Xcode9. The biggest thing that looked different to me was this section that got removed: https://github.com/mozilla-l10n/focusios-l10n/pull/23/files#diff-c6d950ef9857e268acfaebcc2c607122L484

But I will defer to your judgement on that.

### CODE REVIEW

File: en-US/focus-ios.xliff [hunk 1]
```diff
@@ -25,6 +21,10 @@
         <source>This lets you take and upload videos.</source>
         <target>This lets you take and upload videos.</target>
       </trans-unit>
+      <trans-unit id="NSPhotoLibraryAddUsageDescription">
```
Comment (flodolo) [inline en-US/focus-ios.xliff]: I have no clue what this is. Why do we have an empty string?

File: en-US/focus-ios.xliff [hunk 2]
```diff
@@ -456,11 +476,46 @@
...
+        <target>Tracking Protection off</target>
+        <note>text showing the tracking protection is disabled.</note>
+      </trans-unit>
+      <trans-unit id="trackingProtection.socialTrackerLabel">
+        <source>Social trackers</source>
+        <target>Social trackers</target>
+        <note>label for social trackers.</note>
+      </trans-unit>
+      <trans-unit id="trackingProtection.toggleDescription">
+        <source>Disable just for this browsing session if the page isn't behaving as expected.</source>
```
Comment (flodolo) [inline en-US/focus-ios.xliff]: Use proper apostrophes, not straight quotes.

File: en-US/focus-ios.xliff [hunk 3]
```diff
@@ -472,14 +527,14 @@
         <target>$(PRODUCT_NAME)</target>
       </trans-unit>
       <trans-unit id="CFBundleShortVersionString">
-        <source>3.6</source>
-        <target>3.6</target>
+        <source>3.9</source>
```
Comment (flodolo) [inline en-US/focus-ios.xliff]: Did we jump from 3.7 to 3.9?

File: en-US/focus-ios.xliff [hunk 4]
```diff
@@ -2,17 +2,13 @@
...
     <header>
-      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="9.0" build-num="9M214v"/>
     </header>
     <body>
       <trans-unit id="CFBundleName">
         <source>$(PRODUCT_NAME)</source>
         <target>$(PRODUCT_NAME)</target>
       </trans-unit>
-      <trans-unit id="CFBundleShortVersionString">
```
Comment (flodolo) [inline en-US/focus-ios.xliff]: Is this expected? I don't know. Might be a good idea to loop in @st3fan

File: en-US/focus-ios.xliff [hunk 5]
```diff
@@ -490,40 +545,6 @@
...
```
Comment (flodolo) [inline en-US/focus-ios.xliff]: Same as the other removed string. I have no clue if this is OK, that's an iOS development question.

**Comment by flodolo** [comment]
My suggestion would be to use XCode 8 for this cycle, then you should try importing a file in XCode 9, importing it back and see if everything works. It doesn't sound like a good idea to experiment on a real export.

**Comment by boek** [comment]
@flodolo sounds good. Addressing a couple of comments and will have a new PR open shortly.

**Pull request closed without merging**